config_example.json

```diff
@@ -8,15 +8,22 @@
     "log_dir": "/tmp/log",
     "artifact_dir": "/tmp/artifact",
     "task_log_dir": "/tmp/artifact/public/logs",
+    "git_key_repo_dir": "/tmp/key_repo",
+    "base_gpg_home_dir": "/tmp/gpg",
+
+    "my_email": "scriptworker@example.com",
+
     "artifact_expiration_hours": 24,
     "artifact_upload_timeout": 1200,
     "task_script": ["bash", "-c", "echo foo && sleep 19 && exit 1"],
     "task_max_timeout": 1200,
     "valid_artifact_schemes": ["https"],
     "valid_artifact_netlocs": ["queue.taskcluster.net"],
     "valid_artifact_path_regexes": ["^/v1/task/(?P<taskId>[^/]+)(/runs/\\d+)?/artifacts/(?P<filepath>.*)$"],
+
     "verify_chain_of_trust": false,
     "sign_chain_of_trust": false,
+    "gpg_path": null,
```

Comment:
As we're exposing this file to the other instances of scriptworker, I'd make that explicit. I wouldn't bikeshed on that though, 😃